src/components/Footer.js

```diff
@@ -225,8 +225,8 @@ const Footer = () => {
           text: "ef-blog",
         },
         {
-          to: "https://esp.ethereum.foundation",
-          text: "esp",
+          to: "/community/grants",
```

Comment:
I think we still want to link out to esp.ethereum.foundation. Let's add this as an additional link vs. replacing this one.